app/v1r6/controllers/RetrieveEmploymentController.scala

```diff
@@ -64,7 +64,7 @@ class RetrieveEmploymentController @Inject()(val authService: EnrolmentsAuthServ
         employmentId = employmentId
       )
 
-      implicit val desUri: DesUri[RetrieveEmploymentResponse] = DesUri[RetrieveEmploymentResponse](
+      implicit val desUri: IfsUri[RetrieveEmploymentResponse] = IfsUri[RetrieveEmploymentResponse](
```

Comment:
```suggestion
      implicit val ifsUri: IfsUri[RetrieveEmploymentResponse] = IfsUri[RetrieveEmploymentResponse](
```
(we could have just originally called it downstreamUri and we wouldn't have needed to change 😄 )